models/project/board.go

```diff
@@ -224,6 +227,29 @@ func GetBoards(ctx context.Context, projectID int64) (BoardList, error) {
 	return append([]*Board{defaultB}, boards...), nil
 }
 
+func GetBoardsAndCount(ctx context.Context, projectID int64) (BoardList, int64, error) {
+	engine := db.GetEngine(ctx)
+	boards := make([]*Board, 0, setting.UI.IssuePagingNum)
+
+	engine.Where("project_id=? AND `default`=?", projectID, false).OrderBy("Sorting")
```

Comment:
```suggestion
	engine.Where("project_id=?", projectID).OrderBy("Sorting")
```
use opt struct as mentioned for filters ...